### PR TITLE
(nomacs*) Guidelines (fixes #395)

### LINKED ISSUE #395: (nomacs) How should software get updated
Perhaps switch to using Fosshub, as suggested in disqus comment:

https://chocolatey.org/packages/nomacs#comment-2997181741

### PR DESCRIPTION
**Pull request by majkinetor**
This is ready except for proxy (see https://github.com/chocolatey/chocolatey-coreteampackages/issues/395#issuecomment-264851727). 

Waiting for feedback from authors on how to set it up since it uses online installer.

### CODE REVIEW

**Comment by majkinetor** [comment]
I think that this should be pushed now even tho proxy does not work. I added a note for the time being for the online installer package and will wait up for the answer from the vendor. 

The reasons are

- current package doesn't work due to checksum
- it never worked with proxy anyway
- portable package doesn't need proxy and works

If there is no answer for the proxy from authors I can make AHK script for online installer that sets it.

@gep13 @AdmiringWorm

**Comment by majkinetor** [comment]
@gep13 I have a problem to push nomacs as it currently occupies [3.4.1](https://chocolatey.org/packages/nomacs) version and current remove one is 3.4.0.  What do you suggest ? Keep in mind that I would like both nomacs & nomacs.portable which currently doesn't exist to have the same version.

Maybe we should unlist 3.4.1 and I could push 3.4.0.fix for both ?

**Comment by gep13** [comment]
@majkinetor yes, that sounds sensible to me.  I would unlist both the 3.4 and the 3.4.1 packages, and push 3.4.0.

**Comment by majkinetor** [comment]
Sounds great.

**Comment by gep13** [comment]
@majkinetor are you working on this just now?  You want them unlisted just now?

**Comment by majkinetor** [comment]
Yup

**Comment by majkinetor** [comment]
You can see by the labels - its done and waiting for you to finish.

File: automatic/nomacs.portable/nomacs.portable.nuspec [hunk 1]
```diff
@@ -0,0 +1,67 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2011/08/nuspec.xsd">
+  <metadata>
+    <id>nomacs.portable</id>
+    <version>3.4.0.20161205</version>
```
Comment (gep13) [inline automatic/nomacs.portable/nomacs.portable.nuspec]: @majkinetor is the package fix notation required here?  I thought the plan was to try to push 3.4.0?

**Comment by gep13** [comment]
@majkinetor I have unlisted 3.4 and 3.4.1 on chocolatey.org

File: automatic/nomacs.portable/nomacs.portable.nuspec (hunk 1 above)
Comment (gep13) [inline automatic/nomacs.portable/nomacs.portable.nuspec]: Gotcha, no worries.

**Pull request merged**